Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py

```diff
@@ -1891,6 +1891,8 @@ def get_export_list_command(client: Client, args):
 
 
 def main():
+    if is_demisto_version_ge("8.12.0"):
+        return_error("Palo Alto Networks AutoFocus v2 integration is deprecated. Please use Unit 42 Intelligence integration instead.")
```

Comment:
Tested on both 8.11 & 8.12
And seems to work as expected

<img width="999" height="270" alt="Image" src="https://github.com/user-attachments/assets/68c905a6-fb27-40d2-a60f-02f7b1900c7e" />
8.11

<img width="1039" height="281" alt="Image" src="https://github.com/user-attachments/assets/a8d4f4b7-a4a4-48f1-a45b-2f74afe6b39c" />
8.12